### ISSUE TITLE
Node Sass does not yet support your current environment (macOS 10.12.1, Node 7.0.0) [CONTEXT: YARN] {FOLLOW UP TO #1764}

### ISSUE THREAD

**Issue by jasonfb** [sass/node-sass#2701]
NODE: v11.8.0 **installed with yarn** (*not npm*)
node-sass: 4.11.0

working in a Express + CRA app, with a `server/` and `client/` directory (served with concurrently), my problem is only isolated to **development** on Mac oS 10.13 High Sierra.

It happened after a DOWNGRADE of X Code from 10.0-> 9.4.1

first, I was unable `yarn start`, consistently getting this
```
Error: Node Sass does not yet support your current environment: OS X 64-bit with Unsupported runtime (72)
```

I found closed issue https://github.com/sass/node-sass/issues/1764 which seems to have consensus that the fix is `npm rebuild node-sass` is the fix. However, this 1) did not succeed at all for me (it crashed), and 2) I am using **yarn** not npm

- NPM version (`npm -v`):
`6.5.0`

- Node version (`node -v`):
`v11.8.0`

- Node Process (`node -p process.versions`):
```
{ node: '11.8.0',
  v8: '7.0.276.38-node.16',
  uv: '1.25.0',
  zlib: '1.2.11',
  brotli: '1.0.7',
  ares: '1.15.0',
  modules: '67',
  nghttp2: '1.34.0',
  napi: '4',
  llhttp: '1.0.1',
  http_parser: '2.8.0',
  openssl: '1.1.0j',
  cldr: '34.0',
  icu: '63.1',
  tz: '2018e',
  unicode: '11.0' }
```

- Node Platform (`node -p process.platform`):
`darwin`

- Node architecture (`node -p process.arch`):
`x64`

**Comment by jasonfb** [comment]
I ran into this problem after what I Think was caused by an X Code 10.1 -> 9.4 DOWNGRADE (I think that's what caused it). First I had some nasty problems with node-sass, then I blew away my `node_modules` and was then unable to `yarn install` again without serious problems.

then I got through that but got to
```
Error: Node Sass does not yet support your current environment: OS X 64-bit with Unsupported runtime (72)
```

I was able to fix this by switching my system over from (what I think was) brew-installed `node`, which annoyingly is difficult to downgrade, to `nvm` (https://github.com/nvm-sh/nvm)

then, once I made sure I had the right node version, I was able to
`yarn cache clean`

and also `yarn install`

then I ran into a nasty rabbit hole with the dev environment only, which was simply fixed by restarting my system after which my app started booting in dev again.
